File: 0136-single-number/0136-single-number.c
int singleNumber(int* nums, int numsSize) {
     for(int i=0;i<numsSize;i++)
     {  int check=1;
        for(int j=0;j<numsSize;j++)
        {
            if(i!=j && nums[i]==nums[j])
            {
               check=0;
            }
        }
        if(check)
        {
            return nums[i];
        }
     }
     return -1;
}

File: 0136-single-number/0136-single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        Arrays.sort(nums);
        for(int i=1;i<nums.length;i++)
        {
          if(nums[i]!=nums[i-1])
          {
            return nums[i-1];
          }
          else{i+=1;}
        }
    return nums[nums.length-1];
    }
}
